cmd/wksctl/init/init.go

```diff
@@ -173,5 +183,17 @@ func updateManifests(options initOptionType) error {
 }
 
 func initRun(cmd *cobra.Command, args []string) error {
+	if initOptions.version == "" {
+		initOptions.version = version.Version // from main command
+	}
+	bytes, err := ioutil.ReadFile(initOptions.dependencyPath)
+	if err != nil {
+		return err
+	}
+	deps, err := toml.Load(string(bytes))
+	if err != nil {
+		return err
+	}
+	dependencies = deps
```

Comment:
perhaps we can skip the indirection here and just `dependencies, err :=` on the line above?  from a certain perspective, clearing out the `dependencies` variable should be fine (right?) if not even preferable?